test/verify/machineslib.py

```diff
@@ -519,6 +520,7 @@ def checkLibvirtEnabled():
         b.wait_in_text("#virtual-machines-listing thead tr td", "No VM is running")
         b.logout()
 
+        m.execute("systemctl start libvirtd.service")
```

Comment:
This isn't quite the right thing for Fedora ≥ 31 (see issue #13563), but the test will need to be updated for that anyway, so good for now.